### PR TITLE
80 write dashboard page

### PR DESCRIPTION
**Pull request by dorianjagusch**
Dashboard page added. 

- several new components and some modifications to old ones (no need for changes, as I added new params with default values)
- Stats to display are subject to change according to #104 

ToDo: 
- implement stats graph
- dashboard-page responsiveness

### CODE REVIEW

File: backend/Friends/views.py [hunk 1]
```diff
@@ -6,12 +6,13 @@
 from .models import Friend
 from .serializers import FriendInputSerializer, FriendOutputSerializer
 from .friendShipStatus import FriendShipStatus
+
```
Comment (merituulie) [inline backend/Friends/views.py:9]: ?

File: frontend/static/js/pages/profilePage.js [hunk 2]
```diff
@@ -5,10 +5,11 @@ import profileImg from '../components/profileComponents/profileImg.js';
 import profileTitle from '../components/profileComponents/profileTitle.js';
```
Comment (merituulie) [inline frontend/static/js/pages/profilePage.js:5]: mm I'd prefer no changes in this file, only additions if need be!

**Review by merituulie — APPROVED** [review]

File: backend/Friends/views.py [hunk 3]
```diff
@@ -6,12 +6,13 @@
...
 from shared_utilities.decorators import must_be_authenticated, \
 									must_be_body_user_id, \
 									valid_serializer_in_body
 
 class FriendsListView(APIView):
-	@method_decorator(must_be_authenticated)
+	# @method_decorator(must_be_authenticated)
```
[the remark this answers is not in the document]
Comment (dorianjagusch) [reply backend/Friends/views.py:15]: debugging... Sorry

File: frontend/static/js/pages/profilePage.js (hunk 2 above)
Comment (dorianjagusch) [reply frontend/static/js/pages/profilePage.js:5]: Reverted

**Review by merituulie — APPROVED** [review]

**Review by azarSarikhani — APPROVED** [review]

**Review by hussojo — APPROVED** [review]

**Pull request merged**